debian/deepin-graphics-driver-manager.1
.\"                                      Hey, EMACS: -*- nroff -*-
.\" (C) Copyright 2021 ut000205 <huxiaodong@uniontech.com>,
.\"
.TH "deepin-graphics-driver-manager "1" "2021-02-02" "deepin-graphics-driver-manager manpage"
.\" Please adjust this date whenever revising the manpage.
.\"
.\" for manpage-specific macros, see man(7)

.SH NAME
deepin-graphics-driver-manager


CHANGELOG.md
<a name="1.1.9"></a>
## 1.1.9 (2019-06-01)


#### Features

*   allow avoid a solution display in solution list ([dc79289f](https://github.com/linuxdeepin/deepin-graphics-driver-manager/commit/dc79289fe87472c81ef0902b50f326b18b50b5b4))



<a name="1.1.8"></a>
## 1.1.8 (2019-04-10)




<a name="1.1.7.1"></a>
## 1.1.7.1 (2019-03-29)


#### Bug Fixes

*   overlay disable not work when download required packages failed ([58ff63ff](58ff63ff))



<a name="1.1.7"></a>
## 1.1.7 (2019-03-28)


#### Bug Fixes

*   overlay disable failed ([f11659d7](f11659d7))



<a name="1.1.6"></a>
## 1.1.6 (2019-02-26)


#### Bug Fixes

*   stuck in gltest ([5d5d7921](5d5d7921))



<a name="1.1.5"></a>
## 1.1.5 (2019-02-26)




<a name="1.1.4"></a>
## 1.1.4 (2019-01-24)




<a name="1.1.3"></a>
## 1.1.3 (2018-12-29)


#### Bug Fixes

*   rsync conmmand hanlde link-file ([e4964a99](e4964a99))



<a name="1.1.2"></a>
## 1.1.2 (2018-12-21)


#### Bug Fixes

*   focus still on toggle button after reboot/ok button is visible ([636a2b03](636a2b03))



<a name="1.1.1"></a>
## 1.1.1 (2018-11-23)


#### Bug Fixes

*   nvidia glamor start X failed ([b911ee80](b911ee80))

#### Features

*   add ts2desktop ([212716b8](212716b8))
*   add genericName ([04aff9e5](04aff9e5))



<a name="1.1.0"></a>
## 1.1.0 (2018-11-09)




<a name="1.0.9"></a>
## 1.0.9 (2018-10-25)


#### Bug Fixes

*   error handling ([df43fd20](df43fd20))
*   avoid false positive in package detection ([15f854d5](15f854d5))
*   hidpi render svg ([5743c50a](5743c50a))
*   bumblebee resolution identify ([9c87dfd0](9c87dfd0))



<a name="1.0.8"></a>
## 1.0.8 (2018-08-31)


<a name="1.0.7.2"></a>
## 1.0.7.2 (2018-08-31)


#### Bug Fixes

*   prime solution using glvnd drivers ([ea222683](ea222683))
*   bumblebee resolution identify ([51fb17bd](51fb17bd))
*   fix translations ([f88005be](f88005be))
*   add more debug info ([8d6e3264](8d6e3264))
*   merge translations ([afd01891](afd01891))
*   prime solution using glvnd drivers ([6415df62](6415df62))



<a name="1.0.7.1"></a>
### 1.0.7.1 (2018-08-28)


#### Bug Fixes

*   bumblebee resolution identify ([9c87dfd0](9c87dfd0))



<a name="1.0.7"></a>
### 1.0.7 (2018-08-12)


#### Bug Fixes

*   version of resolution style ([74e016e3](74e016e3))



<a name="1.0.6"></a>
### 1.0.6 (2018-08-08)


#### Bug Fixes

*   bumblebee prime install remove ([edc9df72](edc9df72))
*   missing depends package ([0aadf86d](0aadf86d))



<a name="1.0.5"></a>
### 1.0.5 (2018-08-07)


#### Bug Fixes

* **intel:**  uxa sna obscure ([d7ff220d](d7ff220d))



<a name="1.0.4"></a>
### 1.0.4 (2018-07-31)




<a name="1.0.3"></a>
### 1.0.3 (2018-07-31)


#### Features

*   add nvidia prime support ([3562fded](3562fded))

#### Bug Fixes

*   add overlayroot depends ([d74eef8d](d74eef8d))
*   update-initramfs in overlay ([4998e947](4998e947))
*   nvidia card driver install and remove ([7814c1da](7814c1da))
*   can not load driver modules (nvidia_intel: blacklists nvidia) ([cc3c51b6](cc3c51b6))
*   obtain multiple version info ([9ca1220c](9ca1220c))
*   start notify every time when system startup ([ce432c44](ce432c44))
*   intel acceleration mode setting error ([f21f405f](f21f405f))
*   not reboot when user cancel install operate ([33ade111](33ade111))
*   complier warnings ([c129afa7](c129afa7))


<a name="1.0.2"></a>
### 1.0.2 (2018-03-22)

*   Fix translate not loaded on polkit dialog


<a name="1.0.1"></a>
## 1.0.1 (2018-03-16)


#### Features

*   add overlayroot depends ([d74eef8d](d74eef8d))

#### Bug Fixes

*   not reboot when user cancel install operate ([33ade111](33ade111))
*   complier warnings ([c129afa7](c129afa7))



# 1.0
  Release 1.0.

# 0.99
- Internal Preview Version.


translations/deepin-graphics-driver-manager.ts
<?xml version="1.0" ?><!DOCTYPE TS><TS language="en" version="2.1">
<context>
    <name>GLTestWindow</name>
    <message>
        <location filename="../gltest/gltestwindow.cpp" line="9"/>
        <source>Apply</source>
        <translation>Apply</translation>
    </message>
    <message>
        <location filename="../gltest/gltestwindow.cpp" line="10"/>
        <source>Cancel</source>
        <translation>Cancel</translation>
    </message>
    <message>
        <location filename="../gltest/gltestwindow.cpp" line="11"/>
        <source>Please ensure the driver works normally without blurred screen and screen tearing</source>
        <translation>Please ensure the driver works normally without blurred screen and screen tearing</translation>
    </message>
</context>
<context>
    <name>MainWindow</name>
    <message>
        <location filename="../prepare/mainwindow.cpp" line="35"/>
        <source>Switch</source>
        <translation>Switch</translation>
    </message>
    <message>
        <location filename="../prepare/mainwindow.cpp" line="63"/>
        <source>OK</source>
        <translation>OK</translation>
    </message>
    <message>
        <location filename="../prepare/mainwindow.cpp" line="67"/>
        <source>Update</source>
        <translation>Update</translation>
    </message>
    <message>
        <location filename="../prepare/mainwindow.cpp" line="72"/>
        <source>Reboot Now</source>
        <translation>Reboot Now</translation>
    </message>
    <message>
        <location filename="../prepare/mainwindow.cpp" line="77"/>
        <source>Later</source>
        <translation>Later</translation>
    </message>
    <message>
        <location filename="../prepare/mainwindow.cpp" line="82"/>
        <source>Cancel</source>
        <translation>Cancel</translation>
    </message>
    <message>
        <location filename="../prepare/mainwindow.cpp" line="164"/>
        <source>Your hardware is not supported currently, please wait for future version.</source>
        <translation>Your hardware is not supported currently, please wait for future version.</translation>
    </message>
    <message>
        <location filename="../prepare/mainwindow.cpp" line="277"/>
        <source>Switching to the discrete graphics interface may cause a black screen.</source>
        <translation>Switching to the discrete graphics interface may cause a black screen.</translation>
    </message>
    <message>
        <location filename="../prepare/mainwindow.cpp" line="281"/>
        <source>Switching to the integrated graphics interface may cause a black screen.</source>
        <translation>Switching to the integrated graphics interface may cause a black screen.</translation>
    </message>
    <message>
        <location filename="../prepare/mainwindow.cpp" line="419"/>
        <source>Downloading %1</source>
        <translation>Downloading %1</translation>
    </message>
    <message>
        <location filename="../prepare/mainwindow.cpp" line="421"/>
        <source>Downloading the driver for %1, please wait...</source>
        <translation>Downloading the driver for %1, please wait...</translation>
    </message>
    <message>
        <location filename="../prepare/mainwindow.cpp" line="447"/>
        <location filename="../prepare/mainwindow.cpp" line="461"/>
        <source>Sorry, switch failed</source>
        <translation>Sorry, switch failed</translation>
    </message>
    <message>
        <location filename="../prepare/mainwindow.cpp" line="460"/>
        <source>Download failed</source>
        <translation>Download failed</translation>
    </message>
    <message>
        <location filename="../prepare/mainwindow.cpp" line="468"/>
        <source>Download successful</source>
        <translation>Download successful</translation>
    </message>
    <message>
        <location filename="../prepare/mainwindow.cpp" line="470"/>
        <source>Please reboot to test the driver. If no signal, please confirm whether the monitor output port is connected correctly.</source>
        <translation>Please reboot to test the driver. If no signal, please confirm whether the monitor output port is connected correctly.</translation>
    </message>
    <message>
        <location filename="../prepare/mainwindow.cpp" line="472"/>
        <source>Please reboot to test the driver</source>
        <translation>Please reboot to test the driver</translation>
    </message>
</context>
<context>
    <name>Resolution</name>
    <message>
        <location filename="../prepare/resolution.h" line="10"/>
        <location filename="../prepare/resolution.h" line="17"/>
        <source>Integrated Graphics</source>
        <translation>Integrated Graphics</translation>
    </message>
    <message>
        <location filename="../prepare/resolution.h" line="11"/>
        <source>Switching to the discrete graphics interface may cause a black screen</source>
        <translation>Switching to the discrete graphics interface may cause a black screen</translation>
    </message>
    <message>
        <location filename="../prepare/resolution.h" line="12"/>
        <source>Official driver</source>
        <translation>Official driver</translation>
    </message>
    <message>
        <location filename="../prepare/resolution.h" line="13"/>
        <source>Unofficial driver</source>
        <translation>Unofficial driver</translation>
    </message>
    <message>
        <location filename="../prepare/resolution.h" line="14"/>
        <source>Only adaptive to parts of distributions and graphics cards, and it may cause a black screen</source>
        <translation>Only adaptive to parts of distributions and graphics cards, and it may cause a black screen</translation>
    </message>
    <message>
        <location filename="../prepare/resolution.h" line="15"/>
        <source>Poor performance, unstable in some graphics cards</source>
        <translation>Poor performance, unstable in some graphics cards</translation>
    </message>
    <message>
        <location filename="../prepare/resolution.h" line="16"/>
        <source>Switching to the integrated graphics interface may cause a black screen</source>
        <translation>Switching to the integrated graphics interface may cause a black screen</translation>
    </message>
    <message>
        <location filename="../prepare/resolution.h" line="18"/>
        <source>Integrated graphics driver</source>
        <translation>Integrated graphics driver</translation>
    </message>
    <message>
        <location filename="../prepare/resolution.h" line="19"/>
        <source>Bumblebee</source>
        <translation>Bumblebee</translation>
    </message>
    <message>
        <location filename="../prepare/resolution.h" line="20"/>
        <source>Save power to reduce power consumption</source>
        <translation>Save power to reduce power consumption</translation>
    </message>
    <message>
        <location filename="../prepare/resolution.h" line="21"/>
        <source>PRIME</source>
        <translation>PRIME</translation>
    </message>
    <message>
        <location filename="../prepare/resolution.h" line="22"/>
        <source>Best performance, for computers with hybrid graphics</source>
        <translation>Best performance, for computers with hybrid graphics</translation>
    </message>
</context>
<context>
    <name>ResolutionWidget</name>
    <message>
        <location filename="../prepare/resolutionwidget.cpp" line="42"/>
        <source>Driver/Version: %1/%2</source>
        <translation>Driver/Version: %1/%2</translation>
    </message>
</context>
<context>
    <name>main</name>
    <message>
        <location filename="../prepare/main.cpp" line="42"/>
        <source>Graphics Driver Manager</source>
        <translation>Graphics Driver Manager</translation>
    </message>
    <message>
        <location filename="../prepare/main.cpp" line="43"/>
        <source>Graphics Driver Manager is a compact and easy to use graphics driver management tool. It includes graphics card hardware detection, graphics driver installation, graphics driver solution switching,  graphics driver automatic recovery and other functions.</source>
        <translation>Graphics Driver Manager is a compact and easy to use graphics driver management tool. It includes graphics card hardware detection, graphics driver installation, graphics driver solution switching,  graphics driver automatic recovery and other functions.</translation>
    </message>
    <message>
        <location filename="../installer/main.cpp" line="48"/>
        <source>Congratulations, you have switched to %1, please reboot to take effect.</source>
        <translation>Congratulations, you have switched to %1, please reboot to take effect.</translation>
    </message>
    <message>
        <location filename="../installer/main.cpp" line="51"/>
        <source>Cancel</source>
        <translation>Cancel</translation>
    </message>
    <message>
        <location filename="../installer/main.cpp" line="52"/>
        <source>Reboot</source>
        <translation>Reboot</translation>
    </message>
    <message>
        <location filename="../installer/main.cpp" line="90"/>
        <source>Auto restore to %2 after failed to switch to %1</source>
        <translation>Auto restore to %2 after failed to switch to %1</translation>
    </message>
    <message>
        <location filename="../installer/main.cpp" line="95"/>
        <source>Submit Feedback</source>
        <translation>Submit Feedback</translation>
    </message>
    <message>
        <location filename="../installer/main.cpp" line="103"/>
        <source>Updating the driver, please wait...</source>
        <translation>Updating the driver, please wait...</translation>
    </message>
</context>
</TS>